src/test/scala/reactive/ActorTestUtils.scala

```diff
@@ -0,0 +1,30 @@
+package reactive
+
+import scala.reflect.ClassTag
+
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.FlatSpecLike
+import org.scalatest.Matchers
+
+import akka.actor.ActorSystem
+import akka.actor.Status
+import akka.testkit.DefaultTimeout
+import akka.testkit.ImplicitSender
+import akka.testkit.TestKit
+
+class ActorTestUtils extends TestKit(ActorTestUtils.actorSystem())
+  with DefaultTimeout with ImplicitSender
+  with FlatSpecLike with Matchers with BeforeAndAfterAll {
+  override protected def afterAll() = shutdown()
```

Comment:
Maybe add a whiteline to make it a bit clearer where the class declaration ends and the class implementation starts?